packages/fluent-theme/src/fluent/styles/ChoiceGroupOption.styles.ts

```diff
@@ -63,6 +65,9 @@ export const ChoiceGroupOptionStyles = (props: IChoiceGroupOptionStyleProps): Pa
           ]
         }
       },
+      (hasIcon || hasImage) && {
+        paddingTop: 23
```

Comment:
I get that 23 is 1px more from 22px originally but this feels not the solution of adding 1px to several style areas to achieve this. Maybe there is just one place where you can make the change. This will need to be ported to default styles in v7.0 so don't know we want this approach in there.